openmetadata-ui/src/main/resources/ui/src/generated/metadataIngestion/testSuitePipeline.ts

```diff
@@ -15,7 +15,7 @@
  */
 export interface TestSuitePipeline {
     /**
-     * Fully qualified name of the entity to be tested.
+     * Fully qualified name of the entity to be tested, if we're working with a basic suite.
```

Comment:
Are these auto-generated? Because I don't see any schema changes in the PR

Reply:
yes, got auto generated

Reply:
Let's revert the generated file changes.